transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketTest.java

```diff
@@ -19,15 +19,22 @@
 import io.netty.channel.unix.PeerCredentials;
 import io.netty.channel.unix.tests.SocketTest;
 import io.netty.channel.unix.tests.UnixTestUtils;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 import java.io.IOException;
 
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assume.assumeTrue;
 
 public class EpollSocketTest extends SocketTest<LinuxSocket> {
+    @BeforeClass
+    public static void loadJNI() {
+        assumeTrue(Epoll.isAvailable());
```

Comment:
This is already done in 4.0 [1] and the file structure isn't the same anymore so the back port would be a bit messy. Can you clarify what action you want me to take?

[1] https://github.com/netty/netty/blob/4.0/transport-native-epoll/src/test/java/io/netty/channel/unix/SocketTest.java#L31